.travis.yml

```diff
@@ -2,7 +2,8 @@ language: scala
 jdk:
   - oraclejdk8
```

Comment:
I should also add some more tests for collections.   With the changes here, we should in theory be able to create collections of any cql type (including nesting collections (I think)).  I'll do that.